srcpkgs/gitui/template

```diff
@@ -1,14 +1,15 @@
 # Template file for 'gitui'
 pkgname=gitui
-version=0.9.1
+version=0.10.1
 revision=1
 build_style=cargo
+hostmakedepends="python3"
```

Comment:
A new dependency of gitui requires `python3` for building, which is what caused the previous CI failure.